src/components/BookList.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const BookList = ({ books }) => {
+  const getBooks = () => {
```

Comment:
- [ ] BookList should be placed at `src/containers` (directory for container components) since it's connected with Redux.